xdsl/irdl/constraints.py

```diff
@@ -221,6 +221,10 @@ def verify(
         pass
 
 
+# TODO: merge after pyright is updated, and update all constrs
+ANY_ATTR = AnyAttr()
```

Comment:
Why is this necessary?

Reply:
It's not really, it's more of a perf thing, just feels wrong to me to create lots of the same object, but I'm not planning on including it in any of the upcoming PRs, at least not before updating Pyright.